File: Python/3/3.py
# Get input from user
num = int(input("Enter a number: "))

# Store user input num in original 
original = num

# Initialize fact variable as 1
fact = 1

# Multiply num to fact and subtract 1 from num till num > 1
while num > 1:
    fact *= num
    num -= 1

# Print value of original number and its factorial
print("{0}! is {1}".format(original, fact))

File: C/3/3.c
#include <stdio.h>

int i, num, rev, rem;

int main()
{
    // Get input from user
    printf("Enter a number: ");
    scanf("%d", &num);
    i = num;

    // Calculate reverse
    while (i > 0)
    {
        rem = i % 10;
        rev = rev * 10 + rem;
        i = i / 10;
    }

    // Print result
    printf("Reverse of %d is %d", num, rev);

    return 0;
}
